extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -150,9 +178,90 @@ class AmpLightbox extends AMP.BaseElement {
     this.active_ = false;
   }
 
+  /**
```

Comment:
I would add some comments that this is similar to what we do in `scrollable-carousel.js`